src/AddCard.tsx

```diff
@@ -0,0 +1,47 @@
+import { useContext, useState } from 'react'
+import { CardInfoContext } from './context/paymentContext'
+import { Button } from './stories/Button'
+import { CardBox, CardCvc, CardName, CardNumber, CardPassword, CardType, CardDate } from './components'
+import ui from './styles/index.module.css'
+
+type StepProps = {
+  onBack: () => void
+  onNext: () => void
+}
+const AddCard = ({ onBack, onNext }: StepProps) => {
+  const cardInfo = useContext(CardInfoContext)
+  const [isShowModal, setIsShowModal] = useState(false)
+
+  return (
+    <main>
+      <button onClick={() => onBack()}>뒤로가기</button>
+      <h2>1️⃣ 카드 추가</h2>
+      <div className={ui['root']}>
+        <div className={ui['app']}>
+          <h2 className={ui['page-title']}>카드 추가</h2>
+          <CardBox />
+          <CardNumber />
+          <CardDate />
+          <CardName />
+          <CardCvc />
+          <CardPassword />
+          <div className={ui['button-box']}>
+            <Button
+              primary
+              size="small"
+              backgroundColor="#efefef"
+              onClick={() => {
+                if (cardInfo.cardType) onNext()
+                else setIsShowModal((state) => !state)
+              }}
+              label="다음"
+            ></Button>
+          </div>
+        </div>
+      </div>
+      {isShowModal && <CardType setIsShowModal={() => setIsShowModal(false)} />}
```

Comment:
```suggestion
      {isShowModal && <CardType closeModal={() => setIsShowModal(false)} />}
```

setIsShowModal(false)를 실행하려는 함수를 넘기는 prop의 이름이 setIsShowModal 그대로인 부분은 어색한 것 같아요!